website/docs/d/vpc_subnet_v1.html.md

```diff
@@ -59,4 +59,6 @@ the selected subnet.
 
 * `dns_list` - The IP address list of DNS servers on the subnet.
 
-* `dhcp_enable` - DHCP function for the subnet.
\ No newline at end of file
+* `dhcp_enable` - DHCP function for the subnet.
+
+* `subnet_id` - Specifies the subnet ID.
```

Comment:
the description seems identical with the 'id' argument attribute above. In case users get confused here, please change it as what API docs said.

Reply:
@niuzhenguo done